CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Favicon.
```

Comment:
Should be in the `### Added` section and say `- Favicon to documentation pages.`